gnome-initial-setup/eos/hack-sound-client/eos-hack-sound-client.c

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2019 Endless Mobile. Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ *
+ * Written by:
+ *     Cesar Fabian Orccon Chipana <cfoch.fabian@gmail.com>
+ */
+
+#include "eos-hack-sound-client.h"
+
+static void eos_hack_sound_client_initable_iface_init (GInitableIface *iface);
+static void eos_hack_sound_client_finalize            (GObject *object);
+static gboolean eos_hack_sound_client_initable_init (GInitable    *initable,
+                                                     GCancellable *cancellable,
+                                                     GError      **error);
+static void play_sound_cb (GDBusProxy   *proxy,
+                           GAsyncResult *res,
+                           GTask        *task);
+static void stop_sound_cb (GDBusProxy   *proxy,
+                           GAsyncResult *res,
+                           GTask        *task);
+
+typedef struct _EOSHackSoundClientPrivate EOSHackSoundClientPrivate;
+
+struct _EOSHackSoundClient
+{
+  GObject parent_instance;
+  EOSHackSoundClientPrivate *priv;
+};
+
+struct _EOSHackSoundClientPrivate {
```

Comment:
Drop the private structure and put the `GDBusProxy *` into `struct _EOSHackSoundClient`.